yt/yt/ytlib/chunk_client/input_chunk.h

```diff
@@ -140,10 +144,12 @@ class TInputChunk
     i64 GetUncompressedDataSize() const;
     i64 GetCompressedDataSize() const;
 
+    double GetSelectivityFactor() const;
```

Comment:
Let's add some comments about what is GetRowCount(), GetDataWeight(), etc (when is it exact and when approximate). Also, let's comment what is selectivity factor and write somewhere difference between block selectivity factor, column selectivity factor and row selectivity factor.

Reply:
Added a bunch of comments.